dockerfiles/ubuntu/obam/README.md

```diff
@@ -11,7 +11,7 @@ This section defines the step-by-step instructions to build an [Ubuntu](https://
   > The hosted locations of artifacts will be passed as the build arguments when building the Docker image.<br>
   > 1. **WSO2_OB_Accelerator_DIST_URL** - Accelerator location
   > 2. **OB_TRUSTED_CERTS_URL** - Certificate zip archive location
-  > 3. **WSO2_OB_KEYSTORES_URL** - OBAM Keystores folder location
+  > 3. **WSO2_OB_KEYSTORES_URL** - OBAM Keystores folder location (https://github.com/wso2/docker-open-banking/raw/v3.0.0.6/dockerfiles/ubuntu/obam/obam-keystores)
```

Comment:
new tag will be v3.0.0.7

Reply:
Check for other places as well

Reply:
changed it in every necessary places. https://github.com/wso2/docker-open-banking/pull/149/commits/89b83d64b61cb15abe1b1f4ea239a556a28f5fbc